### DISCUSSION TITLE
rebuild ssl

### DISCUSSION THREAD

**Discussion by ychaouche** [pyenv/pyenv#2658] [Q&A]
Hi all,
Is there a way to rebuild 
./build/lib.linux-x86_64-3.10/_ssl.cpython-310-x86_64-linux-gnu.so
w/o starting all over again? 
(I'd like to skip downloading 3.10 and compiling everything else)

**Comment by native-api** [comment]
Technically, you can probably run CPython's Makefile for the specific target.

To avoid downloading again, create a `$PYENV_ROOT/cache` directory. It will cache downloaded stuff.
